rpc/server.go

```diff
@@ -0,0 +1,91 @@
+package rpc
+
+import (
+	"github.com/hashicorp/yamux"
+	log "github.com/sirupsen/logrus"
+	"net"
+	"net/rpc"
+	"net/rpc/jsonrpc"
+	"sync"
+
+	"github.com/thunderdb/ThunderDB/proto"
+)
+
+type ServiceMap map[string]interface{}
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ServiceMap should have comment or be unexported